### ISSUE TITLE
Heatmap x axis issue and annotation issues

### ISSUE THREAD

**Issue by davisriska** [apexcharts/apexcharts.js#1201]
(codepen @ the bottom)
Im making this in angular. 
I set x value as a timestamp. 

I have a formatter
```javascript
public xAxis: ApexXAxis = {
        type: 'datetime',
        labels: {
            formatter: (value, timestamp): string => {
                return moment.unix(Number.parseInt(value)).local().format('HH:mm');
            }
        }
    };
```

Tried setting min, max to force the correct format. That didnt work.
```javascript
this.apexChart.updateOptions({
            xaxis: {
                tickAmount: 12,
                min: this.series[0].data[0].value.date.unix(),
                max: this.series[0].data[this.series[0].data.length - 1].value.date.unix()
            }
        }, false, false, false);
```

This is what happens, time seems all whack. But the tooltip is showing the correct time.
![image](https://user-images.githubusercontent.com/24993105/72154582-b7923480-33b9-11ea-8409-ef80a4b30900.png)

The next problem is with annotations. 
I add annotation with
```javascript
this.apexChart.addPointAnnotation({
                    x: audio.startTsByAggregation(this.aggregation).unix(),
                    y: series.data.find((value) => value.x == audio.startTsByAggregation(this.aggregation).unix()).y,
                    seriesIndex: seriesIndex,
                    label: {
                        borderColor: "#775DD0",
                        style: {
                            color: "#fff",
                            background: "#775DD0"
                        },
                        text: `Audio - ${audio.start_ts.format('YYYY-MM-DD HH:mm')}`
                    }
                });
``` 
What happens is it gets added but in completely wrong place.
![image](https://user-images.githubusercontent.com/24993105/72155079-ee1c7f00-33ba-11ea-815a-b082b283c932.png)
It should be shown in the 30th where the tooltip is shown.
I checked the x, y and the seriesIndex and they are 100% correct.

Any suggestions how I could fix these problems? 

Thanks for this amazing library!

**Comment by davisriska** [comment]
Created a codepen - https://codepen.io/davisriska/pen/mdyLJdO
No idea why annotation dosnt show right away, but if i open "inspect element" it redraws and shows the annotation.

**Comment by davisriska** [comment]
@junedchhipa Any ideas maybe how I could go around this problem while you deal with other issues?

**Comment by junedchhipa** [comment]
Sorry for the late response, but there seems to be an issue with animation and point annotations.
If you disable animations, it shows up on the first render.

I will look into it more what's the cause of the issue.

**Comment by davisriska** [comment]
Seems like the issue is still there. The X axis is broken and the point is still shown in the wrong place.

**Comment by junedchhipa** [comment]
The new version isn't released yet (will be released tonight or tomorrow).
The issue will be fixed in 3.17.1

**Comment by davisriska** [comment]
@junedchhipa Oh, alright. Thanks!
